src/scheduler/node_info.c

```diff
@@ -4889,7 +4934,8 @@ node_state_to_str(node_info *ninfo)
 /**
  * @brief
  *		combine_nspec_array - find and combine any nspec's for the same node
- *		in an nspec array
+ *		in an nspec array.  Because nspecs no longer map to the original chunks
+ *		they came from, seq_num and chk longer have meaning.  They are cleared.
```

Comment:
I think there is a 'no' missing here. It should say seq_num and chk no longer have meaning.